Add banner and content for USS 2023 (#3)
UNIQUE-Students/UNIQUE-Students.github.io: _pages/2023/uss.md

<br>

> # The UNIQUE Student Symposium 2023 is coming soon!

Reviewer: This page still has a lot of content from last year and should be updated before we merge the PR.

Author: Is it about the section "Important dates"?

Reviewer: Not just the dates but some of the descriptions as well, for example:
1. Have we decided whether there will be posters or lighting talks or both?
2. What is the event subtitle? It is still set to last year's (Dynamics of Learning).
3. The accepted posters page should not yet be linked.
4. Is the submission deadline the same this year as well? (May 27)

Author: Added one more commit to fix the main page uss23:
1. changed the title
2. disabled accepted posters for now
3. fixed the important dates
4. in-person for posters and the lightnighting talks

<br>

<div class="card">
<div class="card-image">
    <figure class="image is-5by2">
    <img src="/assets/img/USS2023/banner.png" alt="USS 20230605">
    </figure>
</div>
<div class="card-content">
    <div class="media">
    <div class="media-content">
        <p class="title is-4">UNIQUE Student Symposium 2023: From Brains to Machines and Back: exploring the potential of NeuroAI</p>
    </div>
    </div>

    <div class="content">
    USS is a student-oriented, student-organized conference at the crossroads of neuroscience and AI research. USS 2023 will be held <b>hybrid in Montreal</b> on June 5th and 6th.
    </div>
</div>
</div>

---

## From Brains to Machines and Back: exploring the potential of NeuroAI

##### See the full schedule [here](schedule).
##### Day 1: Monday, 5 June 2023
##### Day 2: Tuesday, 6 June 2023
##### See the speakers [here](speakers).

---

## Important dates

> * Posters and Lightning Talks Registration due: <b>Friday, May 7th, 2023 (in-person)</b>

Reviewer: ```suggestion
* Posters and Lightning Talks Registration due: <b>Friday, May 7th, 2023 (anywhere on Earth)</b>
```
This is about the time zone to submit the posters online. Are you sure that this is the correct deadline?

Author: Yep
